LoginDemo/LearningSelenium/FactoryDesign/FactoryMethodDesign/FMDTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿using LearningSelenium.Utilities;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace LearningSelenium.FactoryDesign.FactoryMethodDesign
+{
+    public class FMDTest
+    {
+        WebDriver driver;
+        Utility utilityObj;
+        ReadConfig config = ReadConfig.GetReadConfig("C:/Kanaga/repo/Login/LoginDemo/LoginDemo/LearningSelenium/FactoryDesign/DataConfig.json");
+
+        [SetUp]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
+            utilityObj = Utility.GetInstance(driver);
+            utilityObj.SetImplicitWait(20);
```

Comment:
You also could place waiting time in config file because any number different from 0 and 1 if it's used in a project should stored as a constant, if it's not suitable to be took out to test/config data

Reply:
wait value is added in config file